XNAControls/XNAControl.cs

```diff
@@ -1034,41 +972,52 @@ protected virtual void ParseControlINIAttribute(IniFile iniFile, string key, str
             case "DistanceFromRightBorder":
                 if (Parent != null)
                 {
-                    ClientRectangle = new Rectangle(Parent.Width - Width - Conversions.IntFromString(value, 0), Y,
-                        Width, Height);
+                    ClientRectangle = new(
+                        Parent.Width - Width - Conversions.IntFromString(value, 0),
+                        Y,
+                        Width,
+                        Height);
                 }
+
                 return;
             case "DistanceFromBottomBorder":
                 if (Parent != null)
                 {
-                    ClientRectangle = new Rectangle(X, Parent.Height - Height - Conversions.IntFromString(value, 0),
-                        Width, Height);
+                    ClientRectangle = new(
+                        X,
+                        Parent.Height - Height - Conversions.IntFromString(value, 0),
+                        Width,
+                        Height);
                 }
+
                 return;
             case "FillWidth":
-                if (Parent != null)
-                {
-                    ClientRectangle = new Rectangle(X, Y,
-                        Parent.Width - X - Conversions.IntFromString(value, 0), Height);
-                }
-                else
-                {
-                    ClientRectangle = new Rectangle(X, Y,
+                ClientRectangle = Parent != null
+                    ? new(
+                        X,
+                        Y,
+                        Parent.Width - X - Conversions.IntFromString(value, 0),
+                        Height)
+                    : new(
+                        X,
+                        Y,
                         WindowManager.RenderResolutionX - X - Conversions.IntFromString(value, 0),
                         Height);
```

Comment:
I think separating every argument, including the one-char Xs and Ys, into their own lines here is just wasting space. I prefer the original.